transactionoutbox-core/src/main/java/com/gruelbox/transactionoutbox/TransactionOutboxImpl.java

```diff
@@ -259,39 +259,41 @@ private void submitNow(TransactionOutboxEntry entry) {
   @Override
   @SuppressWarnings("WeakerAccess")
   public void processNow(TransactionOutboxEntry entry) {
-    try {
-      initialize();
-      var success =
-          transactionManager.inTransactionReturnsThrows(
-              transaction -> {
-                if (!persistor.lock(transaction, entry)) {
-                  return false;
-                }
-                log.info("Processing {}", entry.description());
-                invoke(entry, transaction);
-                if (entry.getUniqueRequestId() == null) {
-                  persistor.delete(transaction, entry);
-                } else {
-                  log.debug(
-                      "Deferring deletion of {} by {}", entry.description(), retentionThreshold);
-                  entry.setProcessed(true);
-                  entry.setLastAttemptTime(Instant.now(clockProvider.get()));
-                  entry.setNextAttemptTime(after(retentionThreshold));
-                  persistor.update(transaction, entry);
-                }
-                return true;
-              });
-      if (success) {
-        log.info("Processed {}", entry.description());
-        listener.success(entry);
-      } else {
-        log.debug("Skipped task {} - may be locked or already processed", entry.getId());
+    entry.getInvocation().withinMDC(() -> {
+      try {
```

Comment:
Mostly whitespace changes. Suggest comparing with whitespace ignored.

Reply:
Could you run a full maven build? It will standardise the formatting.